roles/nginxplus/tasks/modules/install-modules.yml

```diff
@@ -14,5 +14,8 @@
 - import_tasks: install-rtmp.yml
```

Comment:
This might be for a future PR, but I would love to see us remove the `import_tasks` statements, conditions, etc., that we do not use and do not plan to use.